src/store/index.js

```diff
@@ -7,8 +7,9 @@ Vue.use(Vuex)
 
 const newGameState = {
     id: null,
-    state: 'prize',
+    state: 'lobby',
     timer: 1200,
+    playersReady: [0,0],
```

Comment:
We already have a players array.  Maybe throw a (Boolean) ready property on that object instead?  If you'd rather address that in another PR I can merge this one.  Just let me know.

Reply:
ok-- just updated